PaddleCV/image_classification/utils/optimizer.py

```diff
@@ -67,6 +68,33 @@ def cosine_decay_with_warmup(learning_rate, step_each_epoch, epochs=120):
                 fluid.layers.tensor.assign(input=decayed_lr, output=lr)
     return lr
 
+def exponential_decay_with_warmup(learning_rate, step_each_epoch, decay_steps, decay_rate=0.97, warm_up_epoch=5.0):
+    """Applies exponential decay to the learning rate.
+    """
+    global_step = _decay_step_counter()
+    lr = fluid.layers.tensor.create_global_var(
+    shape=[1],
+    value=0.0,
+    dtype='float32',
+    persistable=True,
+    name="learning_rate")
+
+    warmup_epoch = fluid.layers.fill_constant(
+        shape=[1], dtype='float32', value=float(warm_up_epoch), force_cpu=True)
+
+    with init_on_cpu():
+        epoch = ops.floor(global_step / step_each_epoch)
+        with control_flow.Switch() as switch:
```

Comment:
There exists another "fluid.layers.control_flow" in "cosine_decay_with_warmup", feel free to refine it

Reply:
updated.